### PR TITLE
feat(graphql): Enable Enum Values

### PR DESCRIPTION
**Pull request by jordicasanella**
## Summary

changes to enable to validate enum values

# How Has This Been Tested?
 
Unit Test
Install package locally and build fix translator

Screenshot fix-translator checking enum:

image

Screenshot from unit Test:

image

### CODE REVIEW

File: src/main/java/com/valstro/plugin/validator/FieldValidator.java [hunk 1]
```diff
@@ -90,4 +114,8 @@ private boolean typeIsAClassReference(Class<?> clazz) {
         return holder.isExist(clazz.getSimpleName());
     }
 
+    private boolean typeIsAnEnum(Class<?> clazz) {
```
Comment (aditya-valstro) [inline src/main/java/com/valstro/plugin/validator/FieldValidator.java:117]: NIT: Is this helper really needed?

**Review by aditya-valstro — APPROVED** [review]
Just had one small NIT but looks great. I would've liked to see some screenshots in the PR summary that show the logs that the enum recursive check generates. Overall though, nice changes.

File: src/main/java/com/valstro/plugin/validator/FieldValidator.java (hunk 1 above)
Comment (jordicasanella) [reply src/main/java/com/valstro/plugin/validator/FieldValidator.java:117]: I agree with u and adding screenshots now

Comment (jordicasanella) [reply src/main/java/com/valstro/plugin/validator/FieldValidator.java]: I removed the method and moved all the enum validation implementation to his own method

File: src/main/java/com/valstro/plugin/validator/FieldValidator.java [hunk 2]
```diff
@@ -53,31 +59,52 @@ public void validate(Class<?> clazz, Class<?> codeGenClazz) throws MojoFailureEx
         }
     }
 
+    private void checkEnumValues(final Class<?> clazzType) throws MojoFailureException {
+        var codeGen = this.holder.getCodeGenClassFromName(clazzType.getSimpleName());
+        var enumValuesClass = clazzType.getEnumConstants();
+
+        Set<String> enumValues = Arrays.stream(codeGen.getEnumConstants())
```
Comment (jordicasanella) [inline src/main/java/com/valstro/plugin/validator/FieldValidator.java:66]: **note**: I know the existence of EnumSet, however for some reason does not like, this is the reason why I moved to this solution

**Review by ukondoz — APPROVED** [review]

**Pull request merged**